components/DeleteButton/index.js

```diff
@@ -1,17 +1,21 @@
 import styled from "styled-components";
 
-const StyledButton = styled.button`
+const DeleteBtn = styled.button`
+  position: absolute;
+  bottom: 10px;
+  right: 10px;
   border-radius: 8px;
   border: 5px;
-  background-color: #bdbdbd;
+  background-color: #756ab6;
+  color: #fff;
+  font-size: 1.1rem;
   box-shadow: rgba(0, 0, 0, 0.15) 1.95px 1.95px 2.6px;
   font-family: monospace;
-  color: black;
-  height: 25px;
-  margin: 5px;
-  padding: 3px;
+  padding: 10px;
+  height: 40px;
+  width: 100px;
```

Comment:
Whenever possible, I recommend avoiding fixed width and height for your elements. Consider if the user has set their default font size larger than usual. This could result in text overlapping your button. Ideally, your button should expand in accordance with the font size. You could use only padding here.